manifests/job.pp

```diff
@@ -68,7 +68,7 @@
   Optional[String]  $messages            = undef,
   String            $restoredir          = '/tmp/bacula-restores',
   Optional[String]  $sched               = undef,
-  Optional[String]  $priority            = undef,
+  Optional[Variant[Integer,String]]  $priority   = undef,
```

Comment:
Good catch! I wonder if we should not consider switching this to a bare `Optional[Integer]`?

Reply:
Probably.  Although it might be nice to go with the "be forgiving on your input and strict on your output" approach, meaning a custom type that allows Integers and Patterns that look just like an Integer.  That imposes less churn on users of this module, but may limit what might be desired in the future.

Reply:
Agreed!